GSoC25_H/llm_IE/llm_interface.py

```
@@ -18,97 +21,57 @@ class ExtractionResult:
-            response.raise_for_status()
-            
-            result = response.json()
-            return result.get("response", "").strip()
-
-        except requests.exceptions.RequestException as e:
-            print(f"Error during Ollama API request: {e}")
-            return ""
+        # Initialize the shared service
+        self.service = LLMService(shared_config)
 
     def extract_relations(self, sentence: str, prompt: str) -> ExtractionResult:
```

Ruff (0.14.13)

Unused method argument: `sentence`

(ARG002)